platform/src/common/components/Collocation/DeviceStatus/ReportDetailPopup/index.jsx

```diff
@@ -13,9 +65,24 @@ const ReportDetailCard = ({ deviceName, batchId, data, open, closeModal }) => {
       >
         <div className='flex justify-between items-center p-5 border-b border-b-gray-200'>
           <div className='text-black text-base font-medium'>Status summary</div>
-          <button onClick={closeModal} className='btn btn-sm btn-circle btn-ghost'>
-            ✕
-          </button>
+          <span className='flex items-center gap-3'>
+            {/* <DynamicPDFDownloadLink document={<ExportStatusReport batchData={[]} />} fileName="collocation_report.pdf">
+        {({ blob, url, loading, error }) => {
+          setFileBlob(blob);
```

Comment:
Hi @Codebmk , could we remove commented out code? If it is commented, then we do not need it, right?